tests/e2e/scenarios/upgrade-ab/run-test.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "CLOUD_PROVIDER=${CLOUD_PROVIDER}"
+
+if [ -z "$KOPS_VERSION_A" ] || [ -z "$K8S_VERSION_A" ] || [ -z "$KOPS_VERSION_B" ] || [ -z "$K8S_VERSION_B" ]; then
+  >&2 echo "must set all of KOPS_VERSION_A, K8S_VERSION_A, K8S_VERSION_B, K8S_VERSION_B env vars"
+  exit 1
+fi
+
+WORKDIR=$(mktemp)
+
+KOPS_A=${WORKDIR}/kops-${KOPS_VERSION_A}
+wget -O "${KOPS_A}" "https://github.com/kubernetes/kops/releases/download/$KOPS_VERSION_A/kops-$(go env GOOS)-$(go env GOARCH)"
+chmod +x "${KOPS_A}"
+
+KOPS_B=${WORKDIR}/kops-${KOPS_VERSION_B}
+if [[ "${KOPS_VERSION_B}" == "source" ]]; then
+  export KOPS_FEATURE_FLAGS="SpecOverrideFlag,${KOPS_FEATURE_FLAGS:-}"
+  REPO_ROOT=$(git rev-parse --show-toplevel);
+
+  cp "${REPO_ROOT}/bazel-bin/cmd/kops/linux-amd64/kops" "${KOPS_B}"
```

Comment:
side note: it would be great if our scenario scripts could run on more than linux/amd64 but the make task used by prow only builds linux-amd64 builds of the kops cli. Maybe we could extend it to always build the local GOOS / GOARCH as well.

Reply:
Good point.  Could we also just build it here (in the script)?

AFAIK all of our test runners are amd64 anyway (at the moment), so might not be a P0?